CHANGELOG

```diff
@@ -1,7 +1,5 @@
 v3.#.# (MMM’YY)
-  * [feature #1]
-  * [feature #2]
-  * [feature ...]
+  * Update main sidebar
```

Comment:
Since the changelog is now front-facing/customer-facing, we'll benefit more from being more descriptive here. What exactly is updated in the sidebar?